packages/web-app-admin-settings/src/composables/useAdminUsersFlags.ts

```diff
@@ -0,0 +1,20 @@
+import { computed, type ComputedRef } from 'vue'
+import { storeToRefs } from 'pinia'
+import { useCapabilityStore } from '@opencloud-eu/web-pkg'
+
+type Caps = {
+  graph?: { users?: { edit_login_allowed_disabled?: boolean } }
+}
+
+export function useAdminUsersFlags(): {
+  isLoginToggleHidden: ComputedRef<boolean>
+} {
```

Comment:
No need to type that explicitly, TS infers the correct type automatically.

```suggestion
export function useAdminUsersFlags() {
```